lib/src/location_picker_field.dart

```diff
@@ -116,24 +119,29 @@ class __BaseFormFieldState<T> extends State<_BaseFormField<T>> {
                 : effectiveDecoration.suffixIcon);
 
         return ClipPath(
-          clipper: ShapeBorderClipper(shape: effectiveDecoration.border ?? const RoundedRectangleBorder()),
-          child: Material(
-            color: Colors.transparent,
-            child: InkWell(
-              onTap: () => _pick(field),
-              child: MouseRegion(
-                onEnter: (PointerEnterEvent event) => _handleHover(true),
-                onExit: (PointerExitEvent event) => _handleHover(false),
-                child: InputDecorator(
-                  baseStyle: widget.textStyle,
-                  decoration: effectiveDecoration,
-                  isEmpty: widget.isEmpty(field.value),
-                  textAlign: widget.textAlign,
-                  textAlignVertical: widget.textAlignVertical,
-                  isFocused: _isFocused,
-                  isHovering: _isHovering,
-                  expands: widget.expands,
-                  child: widget.display(field.value),
+          clipper: ShapeBorderClipper(
+              shape:
+                  effectiveDecoration.border ?? const RoundedRectangleBorder()),
+          child: Padding(
```

Comment:
why you are adding padding here

Reply:
<img width="342" alt="Screen Shot 2022-04-20 at 8 39 50 PM" src="https://user-images.githubusercontent.com/31937782/164300137-8421ac1d-dafa-4056-8360-f2a0b86f2e4d.png">


as you can see without the padding it has the same behaviour  of text field

Reply:
its not the package responsibility to add spacing between fields another may need a larger space than you or don't need space at all

Reply:
Add some space between fields in this form then select a location in a field and see the result.

Reply:
I should've opened an issue for it with screenshots